src/utils/PublicPathUtils.ts

```diff
@@ -38,7 +58,15 @@ export class PublicPathUtils {
     return <HTMLScriptElement>document.currentScript;
   }
 
+  public static getCoveoScript() {
+    return document.querySelector('.coveo-script');
```

Comment:
Technically, an ID would be more accurate than a class :)

If you want to use a class, I would rather have a `querySelectorAll` and pick the first element that is both `.coveo-script` and a `script` element.

Reply:
Yes it would be more accurate, but adding a class is more flexible. If for some reason they want to use an id on that script and we force them to use this specific ID, there's nothing they can do to have both.

Using `querySelectorAll` and verifying if it's a script seems like a good idea.